azerothcore/azerothcore-wotlk
# fix(DB/MySQL): Support MySQL 8 in create_mysql.sql

## Changes Proposed:
-  With MySQL 8.0 you are not allowed  to create users with GRANT USAGE ON 


## Tests Performed:
- Runs on MySQL 8.0
- @Voxstrasza ran it on MySQL 5.7
- No other tests have been done other than running the SQL



## How to Test the Changes:

Run it and check if you can see all the DB tables after you connect to the DB with these credentials

## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
